src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +512,24 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null or full of
+     * whitespace characters. More efficient that calling
+     * (string).trim().length() == 0 or (string).trim().isEmpty()
+     * because it does not create temporary string whereas trim() does
+     * @param value A string to check
+     * @return true if the value is empty.
+     */
+    public static boolean isOnlyWhitespaces(String value) {
+        if (value == null || value.isEmpty()) {
```

Comment:
After one more review i think name should be `isOnlyWhitespacesOrEmpty`
Ideally user should not read javadoc to know what it is doing. Nuance with null is kind of covered by word Empty.
@Vladlis , @rnveach , please confirm.